src/controllers/product.controller.js

```diff
@@ -0,0 +1,64 @@
+const {user, Product} = require('../../models/index')
+
+exports.postProduct = async (req, res) => {
+    const {name, price, quantity, user_id} = req.body
```

Comment:
in verify u parse userid to req.id , and this user_id should get from req. id not from req.body